### PR TITLE
add new tick events

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@dvoytenko PTAL

**Comment by dvoytenko** [comment]
LGTM

File: TICKEVENTS.md [hunk 1]
```diff
@@ -25,8 +25,11 @@ As an example if we executed `perf.tick('label')` we assume we have a counterpar
 | Name                | id                | Description                        |
 ----------------------|-------------------|------------------------------------|
 | Install Styles      | `is`              | Set when the styles are installed. |
+| End Install Styles  | `e_is`            | Set when the styles are done installing. |
```
Comment (jridgewell) [inline TICKEVENTS.md]: Is this one being added?

Comment (erwinmombay) [inline TICKEVENTS.md]: yeah we've had it for a while now but it isnt getting processed, so fixing that

**Comment by erwinmombay** [comment]
@jridgewell can you take another look at this since @dvoytenko is out? tests should be passing now.

**Comment by jridgewell** [comment]
LGTM.

File: src/service.js [hunk 2]
```diff
@@ -39,8 +40,11 @@ let ServiceHolderDef;
  * @return {!Object} The service.
  */
 export function getExistingServiceForWindow(win, id) {
-  return dev().assert(
-      win.services[id] && win.services[id].obj, `${id} service not found.`);
+  const exists = win.services && win.services[id] && win.services[id].obj;
+  if ((getMode().localDev || getMode().test) && !exists) {
```
Comment (erwinmombay) [inline src/service.js]: @jridgewell added this. let me know if worthwhile or not

Comment (erwinmombay) [inline src/service.js]: disregard. removed this and fixed actual assertion to check every access to get a better error message with stack trace

**Comment by erwinmombay** [comment]
changed a bit of testing code to get this green

**Pull request merged**